packages/dd-trace/src/remote_config/scheduler.js

```diff
@@ -17,7 +17,7 @@ class Scheduler {
   runAfterDelay (interval = this._interval) {
     this._timer = setTimeout(this._callback, interval, () => this.runAfterDelay())
 
-    this._timer.unref && this._timer.unref()
+    this._timer.unref?.()
```

Comment:
This seems like a mistake - The `unref` function was added in Node.js v0.9.1, so why can't we just:

```suggestion
    this._timer.unref()
```